app/views/duties/_open_drop_modal.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="modal-content">                             
+  <div class="modal-header">
+    <h5 class="modal-title" id="exampleModalLabel">Drop</h5>
+    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+  <div class="modal-body">
+    <%= form_tag(drop_duties_path, method: :post) do %>
+      <% @drop_duty_list.each do |d| %>
+        <%= check_box_tag "duty_id[]", d.id %>
+        <%= d.timeslot.time_range.start_time.strftime('%H:%M') + " - " + d.timeslot.time_range.end_time.strftime('%H:%M') %>
```

Comment:
Probably can abstract this line into helpers. Look out `app/helpers/duties_helper.rb`